pom.xml

```diff
@@ -17,8 +17,8 @@
         <disableTestInjection>true</disableTestInjection>
         <hamcrest.version>2.2</hamcrest.version>
         <java.level>8</java.level>
-        <jackson.version>2.11.2</jackson.version>
-        <jenkins.version>2.249.1</jenkins.version>
+        <jackson.version>2.13.0</jackson.version>
+        <jenkins.version>2.289.1</jenkins.version>
```

Comment:
As mentioned duing our chat, I think it would be good to have a comment here so we update the `io.jenkins.tools.bom` at the same time.